esphome/components/esp32/preferences.cpp

```diff
@@ -65,6 +66,8 @@ class ESP32PreferenceBackend : public ESPPreferenceBackend {
     if (err != 0) {
       ESP_LOGV(TAG, "nvs_get_blob('%s') failed: %s", key.c_str(), esp_err_to_name(err));
       return false;
+    } else {
+      ESP_LOGV(TAG, "nvs_get_blob: key: %s, len: %d", key.c_str(), len);
```

Comment:
```suggestion
      ESP_LOGVV(TAG, "nvs_get_blob: key: %s, len: %d", key.c_str(), len);
```
Same here (for the failure above verbose is better IMO because then you know no data was read as a developer when developing a component)